osmflatc/src/strings.rs

```diff
@@ -1,9 +1,63 @@
 use ahash::AHashMap;
-use inlinable_string::{InlinableString, StringExt};
+
+#[derive(Debug, Clone, Copy)]
+struct TerminatedStringPtr {
```

Comment:
Sad, that `&CStr` is 16 bytes

Reply:
Yes, I was looking to CStr as an abstraction.

Best case I really like to use less than 8 bytes (5 are enough): if I could just store an index into `data`, and the Hashmap would provide `data` as a `context` to Hash/PartialEq...